ecosystem/scenarios/utils.py

```
@@ -0,0 +1,260 @@
+            "__file__": str(setup_py),
+            "__name__": "__main__",
+            "setup": mock_setup,
+            "setuptools": setuptools_mock,
+            "find_packages": mock_find_packages,
+            "sys": sys,
+            "os": __import__("os"),
+            "open": open,
+        }
+
+        # Execute setup.py
+        with open(setup_py, "r") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)